packages/provider/src/tasks/detection/getBotScore.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021-2024 Prosopo (UK) Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import getBotScoreFromPayload from "./decodePayload.js";
+
+export const getBotScore = async (payload: string) => {
```

Comment:
This function throws when I run a local provider
```
privateKey
---------------- 
Error: error:1E08010C:DECODER routines::unsupported
    at node:internal/crypto/cipher:79:12
    at vXHwNh (file:///home/user/captcha-private/captcha/packages/provider/dist/tasks/detection/decodePayload.js:125:323)
    at gKTAGWo (file:///home/user/captcha-private/captcha/packages/provider/dist/tasks/detection/decodePayload.js:190:578)
    at getBotScore (file:///home/user/captcha-private/captcha/packages/provider/dist/tasks/detection/getBotScore.js:4:32)
    at file:///home/user/captcha-private/captcha/packages/provider/dist/api/captcha.js:168:36
    at Layer.handle [as handle_request] (/home/user/captcha-private/captcha/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/user/captcha-private/captcha/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/home/user/captcha-private/captcha/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/home/user/captcha-private/captcha/node_modules/express/lib/router/layer.js:95:5)
    at /home/user/captcha-private/captcha/node_modules/express/lib/router/index.js:284:15 {
  library: 'DECODER routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_UNSUPPORTED'
}
```

https://stackoverflow.com/questions/75959563/node-js-err-ossl-evp-unsupported-error-when-running-npm-run-start ?

Reply:
Yep, running with legacy SSL provider fixes it. This will need to be tested on provider VPS's as they will likely hit the same error.

Reply:
```
openssl --version
OpenSSL 3.3.1 4 Jun 2024 (Library: OpenSSL 3.3.1 4 Jun 2024)
```

Reply:
Resolved/to resolve on slack